Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM cgr.dev/chainguard/go:latest as gobuild
```

Comment:
Maybe use the go1.21 image equivalent, no need for latest :)

Reply:
dynamically using version from gomod